account_invoice_auto_send_mail/models/account_journal.py

```diff
@@ -0,0 +1,20 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    invoice_mail_template_id = fields.Many2one(
```

Comment:
Hello, we we really need to have the email template per Journal? Is this the need?
I would prefer if we could minimize the complexity of account.journal, which is an important object, just for this feature.

Possible to put it in Configuration window instaead? Or if there is really need to be at journal level, may be a separated table to keep the info?

For this point, not sure if there is a common practice @OCA/accounting-maintainers

Reply:
Hello, several weeks ago I had the addon developed regarding ir.config.parameter but it did not allow configuring which invoices you wanted to send and / or if we wanted to send different emails (to solve this "problem" I thought that defining the fields at account_journal level would be good idea).